amr-wind/incflo.cpp

```diff
@@ -309,15 +310,17 @@ void incflo::Evolve()
     }
 }
 
-void incflo::do_advance()
+void incflo::do_advance(const int inonlin)
 {
     if (m_sim.has_overset()) {
         m_ovst_ops.pre_advance_work();
     }
-    if (m_prescribe_vel) {
+    if (m_prescribe_vel && inonlin == 0) {
         prescribe_advance();
     } else {
-        advance();
+        amrex::Print() << "Iteration " << inonlin
+                       << " in advection iteration loop" << std::endl;
```

Comment:
does this get printed only in the amr-wind standalone case, and not when running exawind-driver? Just wanted to check, related to your comment on distinguishing the two.